subsys/llext/llext_load.c

```diff
@@ -365,6 +365,29 @@ static int llext_map_sections(struct llext_loader *ldr, struct llext *ext)
 				continue;
 			}
 
+			/*
+			 * Exported symbols region can also overlap
+			 * with rodata.
+			 */
+			if (i == LLEXT_MEM_EXPORT || j == LLEXT_MEM_EXPORT) {
+				continue;
```

Comment:
I found this using a (fairly complex) example of my own, that is currently out-of-tree; it might not be necessary to include this to pass the current unit tests.

Reply:
LGTM, makes sense to keep it. I have a similar exception in one of my branches as well :slightly_smiling_face: